typescript/cli/src/deploy/warp.ts

```diff
@@ -255,7 +255,7 @@ async function executeDeploy(params: DeployParams) {
 
   const [contractsFilePath, tokenConfigPath] = prepNewArtifactsFiles(outPath, [
     { filename: 'warp-route-deployment', description: 'Contract addresses' },
-    { filename: 'warp-ui-token-config', description: 'Warp UI token config' },
+    { filename: 'warp-config', description: 'Warp config' },
```

Comment:
I'm in favor of this rename but we'll need to update the docs too here: https://docs.hyperlane.xyz/docs/guides/deploy-warp-route#2-deployment

Reply:
https://github.com/hyperlane-xyz/v3-docs/pull/72